metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -71,6 +71,8 @@
 ENV_METADATA_HOST_DEPRECATED = "GMS_HOST"
```

Comment:
Seems we are using GMS_HOST?

So why are we adding new environment variables?

Reply:
This is a change for backward compatibility to avoid breaking stuff for now. Once the changes are out will remove these from docker, k8s and CLI

Reply:
hmmm ok